lib/src/model/container.dart

```diff
@@ -46,15 +45,15 @@ abstract class Container extends ModelElement with TypeParameters {
       element is ClassElement && (element as ClassElement).isMixin;
 
   @mustCallSuper
-  Iterable<ModelElement> get allModelElements => quiver.concat([
-        instanceMethods,
-        instanceFields,
-        instanceOperators,
-        instanceAccessors,
-        staticFields,
-        staticAccessors,
-        staticMethods,
-      ]);
+  Iterable<ModelElement> get allModelElements => [
```

Comment:
The major advantage of `quiver.concat` is lazy evaluation and not having to build an entire list.   Are you certain there is no performance difference on complex packages?   This getter and others like it are used quite frequently.

Reply:
Doing some benchmarks, the laziness can help in cases where we truly do use things lazily and use some values, but in most cases we just end up using every value anyway in which case the spread operator ends up being faster pretty much every time.

Perhaps the solution in this case is to cache this result similar to the others.